cranelift/codegen/src/isa/zkasm/inst/emit.rs

```diff
@@ -460,10 +460,7 @@ impl MachInstEmit for Inst {
                 let rd = allocs.next(rd.to_reg());
                 // TODO(nagisa): this wants to conditionally use the `SUB` instruction instead (if
                 // the negation is for 64 bits)
```

Comment:
Not really. Just tried to run
```
 9223372036854775803n - 3n => A
 ```
 Everything was OK. The only note here is that both numbers should have ``n`` or both don't have. (Runned it on our current 256-bit prover)